server/src/main/kotlin/com/sangyoon/parkingpass/parking/repository/SupabaseGateDeviceRepository.kt

```
@@ -49,5 +49,40 @@ class SupabaseGateDeviceRepository(
+    }
+
+    override suspend fun delete(id: Long): Boolean {
+        return try {
+            supabase.from("gate_device")
+                .delete {
+                    filter {
+                        eq("id", id)
+                    }
+                }
+            true
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)